### PR TITLE
Add additional query parameters from vercel

### PR DESCRIPTION
**Pull request by derrickmehaffy**
I'm sure not all of these are useful, but regardless in case other needed them I added them.

Personally I just need the target and app vars and none of the dates but I figured since Vercel offered them might as well put them there. There is also an undocumented limit which I didn't figure needed documentation but just in case it's there.

### CODE REVIEW

File: index.mjs [hunk 1]
```diff
@@ -8,16 +8,34 @@ async function run() {
...
 
-    core.info(`Retrieving deployment preview for ${teamId}/${projectId} ...`)
+    core.info(`Retrieving deployment preview for ${vTeamId}/${vProjectId} ...`)
     const { url, state } = await getDeploymentUrl(
       vercelToken,
       githubRepo,
       githubBranch,
-      teamId,
-      projectId
+      vApp,
```
Comment (PatrickHeneise) [inline index.mjs:28]: Can we put optional parameters into "options", then the function signature doesn't need to change that often.

**Review by PatrickHeneise — CHANGES_REQUESTED** [review]
Thanks @derrickmehaffy! I left a small comment.

File: index.mjs (hunk 1 above)
Comment (PatrickHeneise) [reply index.mjs:28]: Correct, that should give more flexibility with optional parameters, what do you think?

Comment (derrickmehaffy) [reply index.mjs:28]: Makes ense @PatrickHeneise I'll try and get to it this week or my branch should allow you to push new commits too if you wanted to bypass me. I've implemented this in the workflow I needed it in via my fork but I'd like to swap back to yours instead once I get some time to finish this PR.

**Pull request merged**